test/item_parser_test.rb

```diff
@@ -4,6 +4,6 @@
 
 class ItemParserTest < MiniTest::Test
   def test_it_exists
-    assert ItemParser.new('test/support/sample_items.csv')
+    assert ItemParser.new('test/support/items.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.